File: corpus/2017q1/_p2q2_ok/_check/F;0275.java

import java.util.Scanner;
public class CristianeAkemiMarquesEnobe_q2_cardapio(2) {
    public static void main (String[]args){
        double compra=0.00
        int codigo[] ={300,301,302,303,304,305};
        string opcao []={Pao_de_queijo, Bauru, Cachorro_quente, Cheeseburguer, suco, refrigerante};
        double preco [] = {1.30 ,1.40, 1.50, 2.20, 1.00};

        ImprimirCardapio (codigo, opcao, preco);

    }
   public static void ImprimirCardapio (int codigo[], string opcao [], double preco){
       for (int i=0; i<6; i++){
            System.out.print (codigo[i] + " ");
            System.out.print (opcao[i] + " ");
            System.out.print (preco[i] + " ");
        }
   } 
}

File: corpus/2017q1/_p2q2_ok/_check/F;0275.py
double compra=0.00
codigo = [300,301,302,303,304,305]
opcao = [Pao_de_queijo, Bauru, Cachorro_quente, Cheeseburguer, suco, refrigerante]
preco = [1.30 ,1.40, 1.50, 2.20, 1.00]
ImprimirCardapio (codigo, opcao, preco)
def ImprimirCardapio(int codigo[], string opcao [], double preco):
    for i=0 in range(0, 6):
        print (codigo[i] + " ")
        print (opcao[i] + " ")
        print (preco[i] + " ")